crates/zkvm-interface/src/reports.rs

```diff
@@ -10,13 +10,15 @@ pub struct ProgramExecutionReport {
     pub total_num_cycles: u64,
     /// Region-specific cycles, mapping region names (e.g., "setup", "compute") to their cycle counts.
     pub region_cycles: IndexMap<String, u64>,
+    /// Execution duration.
+    pub execution_duration: Duration,
 }
 
 impl ProgramExecutionReport {
     pub fn new(total_num_cycles: u64) -> Self {
```

Comment:
Not that sure this ctor makes sense now that prob all impls can support setting the `elapsed_duration`.

I can also add it as a ctor parameter and use it that way. LMK

Reply:
We can add elapsed_duration too, but not a blocker for this PR. Using something like the builder pattern in the future, is likely the better API I think